scripts/sync-kernel.sh

```diff
@@ -47,6 +47,7 @@ PATH_MAP=(									\
 	[tools/include/uapi/linux/netlink.h]=include/uapi/linux/netlink.h	\
 	[tools/include/uapi/linux/pkt_cls.h]=include/uapi/linux/pkt_cls.h	\
 	[tools/include/uapi/linux/pkt_sched.h]=include/uapi/linux/pkt_sched.h	\
+	[Documentation/bpf/libbpf]=src/docs	\
 )
 
 LIBBPF_PATHS="${!PATH_MAP[@]} :^tools/lib/bpf/Makefile :^tools/lib/bpf/Build :^tools/lib/bpf/.gitignore :^tools/include/tools/libc_compat.h"
```

Comment:
I'm currently getting this when running sync script:

```
...
Comparing list of files...
--- /tmp/tmp.0XYbOenSZb/linux-view.ls   2021-08-11 12:03:46.248692923 -0700
+++ /tmp/tmp.0XYbOenSZb/github-view.ls  2021-08-11 12:03:46.250692920 -0700
@@ -17,10 +17,12 @@
 src/btf.c
 src/btf.h
 src/btf_dump.c
-src/docs/libbpf.rst
-src/docs/libbpf_api.rst
-src/docs/libbpf_build.rst
-src/docs/libbpf_naming_convention.rst
+src/docs/.gitignore
+src/docs/api.rst
+src/docs/conf.py
+src/docs/sphinx/Makefile
+src/docs/sphinx/doxygen/Doxyfile
+src/docs/sphinx/requirements.txt
 src/gen_loader.c
 src/hashmap.c
 src/hashmap.h
```

Please adjust LIBBPF_VIEW_EXCLUDE_REGEX and LINUX_VIEW_EXCLUDE_REGEX accordingly to make any differences go away.

Reply:
I can't reproduce this issue running it how you told me to, I'm not sure what the output is saying, what causes this?